src/pe_reports/data/db_query.py

```diff
@@ -0,0 +1,193 @@
+
+
+def query_hibp_view(conn, org_uid, start_date, end_date):
+    """Query hibp table."""
+    try:
+        sql = """SELECT * FROM vw_breach_complete
+        WHERE organizations_uid = %(org_uid)s
+        AND modified_date BETWEEN %(start_date)s AND %(end_date)s"""
+        df = pd.read_sql(
+            sql,
+            conn,
+            params={"org_uid": org_uid, "start_date": start_date, "end_date": end_date},
+        )
+        return df
+    except (Exception, psycopg2.DatabaseError) as error:
+        logging.error(f"There was a problem with your database query {error}")
+    finally:
+        if conn is not None:
+            close(conn)
+
+
+def query_domMasq(conn, org_uid, start_date, end_date):
+    """Query domain masquerading table."""
+    try:
+        sql = """SELECT * FROM dnstwist_domain_masq
+        WHERE organizations_uid = %(org_uid)s
+        AND date_observed BETWEEN %(start_date)s AND %(end_date)s"""
+        df = pd.read_sql(
+            sql,
+            conn,
+            params={
+                "org_uid": org_uid,
+                "start_date": start_date,
+                "end_date": end_date,
+            },
+        )
+        return df
+    except (Exception, psycopg2.DatabaseError) as error:
+        logging.error(f"There was a problem with your database query {error}")
+    finally:
+        if conn is not None:
+            close(conn)
+
+
+def query_shodan(conn, org_uid, start_date, end_date, table):
+    """Query Shodan Table."""
+    try:
+        sql = """SELECT * FROM %(table)s
+        WHERE organizations_uid = %(org_uid)s
+        AND timestamp BETWEEN %(start_date)s AND %(end_date)s"""
+        df = pd.read_sql(
+            sql,
+            conn,
+            params={
+                "table": AsIs(table),
+                "org_uid": org_uid,
+                "start_date": start_date,
+                "end_date": end_date,
+            },
+        )
+        return df
+    except (Exception, psycopg2.DatabaseError) as error:
+        logging.error(f"There was a problem with your database query {error}")
+    finally:
+        if conn is not None:
+            close(conn)
+
+
+def query_darkweb(conn, org_uid, start_date, end_date, table):
+    """Query Dark Web Table."""
+    try:
+        sql = """SELECT * FROM %(table)s
+        WHERE organizations_uid = %(org_uid)s
+        AND date BETWEEN %(start_date)s AND %(end_date)s"""
+        df = pd.read_sql(
+            sql,
+            conn,
+            params={
+                "table": AsIs(table),
+                "org_uid": org_uid,
+                "start_date": start_date,
+                "end_date": end_date,
+            },
+        )
+        return df
+    except (Exception, psycopg2.DatabaseError) as error:
+        logging.error(f"There was a problem with your database query {error}")
+    finally:
+        if conn is not None:
+            close(conn)
+
+
+def query_darkweb_cves(conn, table):
+    """Query Dark Web CVE Table."""
```

Comment:
Use capitalization consistent with other docstrings:
```suggestion
    """Query Dark Web CVE table."""
```